README.md

```diff
@@ -124,7 +124,7 @@ if creds, ok := runtimeConfig.Credentials("database"); ok {
 
 The return value of `Credentials()` is a `Credential` struct, which includes the appropriate user, password, host, database name, and other pertinent information.  See the [Service documentation](https://docs.platform.sh/configuration/services.html) for your service for the exact structure and meaning of each property.  In most cases that information can be passed directly to whatever other client library is being used to connect to the service.
 
-If `ok` is false it means the specified relationship was not defined so no credentisl are available.
+If `ok` is false it means the specified relationship was not defined so no credentials are available.
 
 ## Formatting service credentials
```

Comment:
And totally rewrite this section. :smile:

Reply:
TBA